build/cqs.tsv

```diff
@@ -1,3 +1,4 @@
+asm:asm	3.1	compile
```

Comment:
I am not depending on it directly as far as I know, I will look into why this was added